specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/webapplicationfirewall.json

```diff
@@ -406,6 +406,13 @@
         "managedRuleSets"
       ],
       "properties": {
+        "allowlists": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AllowlistEntry"
```

Comment:
For composite array, provide:

"x-ms-identifiers": [ "xyz" ]

, if applicable

https://azure.github.io/autorest/extensions/#x-ms-identifiers

Reply:
There isn't any identifying property for allowlists. Is it a requirement to add this x-ms-identifiers or is it optional?

Reply:
optional. If you feel not needed, you may leave it..